### PR TITLE
docs: add activities to swagger v2 docs

### PR DESCRIPTION
**Pull request by sborrazas**

### CODE REVIEW

**Review by thepiwo — APPROVED** [review]

File: docs/swagger_v2/activities.spec.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+          - fee_lock_name
+          - fee_refund_name
+          - fee_spend_name
+          - reward_oracle
+          - fee_lock_name
+      ref_tx_hash:
+        description: The transaction hash of a transaction related to the internal transfer
+        type: string
+  Activity:
+    description: Name Transfer
```
Comment (jyeshe) [inline docs/swagger_v2/activities.spec.yaml:69]: More generic description needed to contemplate the other event types.

File: priv/static/swagger/swagger_v2.yaml [hunk 2]
```diff
@@ -26,6 +26,51 @@ components:
         pattern: (txi|gen):\d+(-\d+)?
         type: string
   schemas:
+    Activity:
+      description: Name Transfer
```
Comment (jyeshe) [inline priv/static/swagger/swagger_v2.yaml:30]: same

File: docs/swagger_v2/activities.spec.yaml (hunk 1 above)
Comment (sborrazas) [reply docs/swagger_v2/activities.spec.yaml]: Done

File: priv/static/swagger/swagger_v2.yaml (hunk 2 above)
Comment (sborrazas) [reply priv/static/swagger/swagger_v2.yaml]: Just as a reminder, this file is auto-generated. If it differs from the previous one the CI will fail

**Review by jyeshe — APPROVED** [review]
Please rebase with last-block-validate-range branch or master after it's merge.

**Pull request merged**